src/app/fsat/fsat-report/results-summary/results-summary.component.html

```diff
@@ -135,7 +135,38 @@ <h4 class="show-print">Report Results</h4>
           [ngClass]="{'selected-modification': index == selectedModificationIndex}">
           {{modification.fsat.outputs.motorPower
           | number:'1.0-1'}}</td>
+      </tr>  
+      
+      
+
+    <!-- ^^^ Co2 section should mimick how annual energy and energy savings display their data below. 
+      You'll need to set these values in the fsat-report since there's no get diff here-->
+      <tr class="callout">
+        <th>Annual CO2 Emissions (tonne CO<sub>2</sub>)</th>
+        <th [ngClass]="{'selected-modification': selectedModificationIndex == -1}">{{fsat.outputs.co2EmissionsOutput |
+          number:'1.1-1'}}</th>
+        <th *ngFor="let modification of fsat.modifications; let index = index;"
+          [ngClass]="{'selected-modification': index == selectedModificationIndex}">
+          <span *ngIf="modification.fsat.outputs.co2EmissionsOutput">{{modification.fsat.outputs.co2EmissionsOutput |
+            number:'1.1-1'}}</span>
+          <span *ngIf="!modification.fsat.outputs.co2EmissionsOutput">&mdash;</span>
+        </th>
+      </tr>
+      <tr class="callout">
+        <th>Annual CO2 Emissions Savings (tonne CO<sub>2</sub>)</th>
+        <th [ngClass]="{'selected-modification': selectedModificationIndex == -1}">&mdash;</th>
+        <th *ngFor="let modification of fsat.modifications; let index = index;"
+          [ngClass]="{'positive-savings': modification.fsat.outputs.percent_annual_savings > 10, 'selected-modification': index == selectedModificationIndex}">
+          <span
+            *ngIf="modification.fsat.whatIfScenario == true">
+            {{modification.fsat.outputs.annualSavings | sigFigs:'3'}}</span>
```

Comment:
This should be emissions savings not annual savings..

baseline co2 emissions - modificaiton co2 emissions